src/home-page/components/GridContainer.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+/* Grid container for the main page */
+export const GridContainer = styled('div')`
+    max-width: 960px;
+    width: 80%;
+    display: grid;
+    margin: auto; 
+    margin-top: 2em;
+    grid-template-columns: repeat(auto-fit, minmax(min(350px, 100%), 1fr));
+    grid-gap: 2em;
+    justify-items: center;
```

Comment:
Maybe `justify-items: stretch;` would be a better fit for our needs? :)

Reply:
Hmm... I used center because when it is too narrow for two elements it is still too wide for one - in my opinion. But i can add center to media queries for tablets, and for desktop and phone it can be stretch or we can change it when we will see how it works with actual card inside.